mkdocs.yml

```diff
@@ -318,24 +332,28 @@ nav:
         - nautobot.apps.ui: "code-reference/nautobot/apps/ui.md"
         - nautobot.apps.urls: "code-reference/nautobot/apps/urls.md"
         - nautobot.apps.views: "code-reference/nautobot/apps/views.md"
-      - Migration Guides: "development/apps/migration/from-v1.md"
+      - Migration Guides:
+        - "development/apps/migration/from-v1.md"
```

Comment:
I think `development/apps/migration/from-v1.md` should be an entry in this section instead of being the index so it shows up in the menu. This is how it looks now:

![image](https://github.com/nautobot/nautobot/assets/75227981/7a633434-eda6-4c77-8db1-9c9924958e8c)

Reply:
Perhaps?

```suggestion
        - From Nautobot v1: "development/apps/migration/from-v1.md"
```

Reply:
Looks good to me!